src/editorManager.js

```diff
@@ -148,8 +148,9 @@ class EditorManager {
     }
 
     const { row, col } = this.instance.selection.selectedRange.current().highlight;
+    const visualColumn = this.instance.renderedToVisualColumn(col);
```

Comment:
Maybe we can use `Selection` here instead of `Highlight`? We would have index already translated to `visual`. This will work but may be an unnecessary op if we have it already translated.